### PR TITLE
Ignore generated files

### PR DESCRIPTION
**Pull request by connorimes**
Git should ignore files generated during build.

### CODE REVIEW

**Comment by richards12** [comment]
Great minds think alike.  I committed a similar change earlier today in my local repo, but hadn't yet had time to make the pull request.

Before I can accept this, I'd like to request two changes:

1.  *.hh is too broad.  However, git_hash.hh and git_vers.hh are perfectly reasonable.
2. You probably also want to ignore .depend.bak.

If you'll make these two changes I'll accept the pull request.  Thanks for your interest in Quicksilver

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,8 @@
 *.exe
 *.out
 *.app
+
+# Project-specific
+.depend
+*.hh
```
Comment (richards12) [inline .gitignore]: Too broad.  Please just git_hash.hh and git_vers.hh

File: .gitignore [hunk 2]
```diff
@@ -30,3 +30,8 @@
 *.exe
 *.out
 *.app
+
+# Project-specific
+.depend
```
Comment (richards12) [inline .gitignore:35]: .depend.bak too.

**Review by richards12 — CHANGES_REQUESTED** [review]

**Comment by connorimes** [comment]
OK updated and rebased.  I didn't originally include `.depend.bak` or `load.map`, `*.core`, or `*.optrpt` since I didn't see where they were being created.  Should those also be included?

**Review by richards12 — APPROVED** [review]

**Comment by richards12** [comment]
.depend.bak is generated by makedepend.  The other files you mention are left-overs from some old build configurations that we deleted from the Makefile for simplicity.  There is no need to add them to the .gitignore file.

Thanks for your interest (and help) in Quicksilver.

**Comment by connorimes** [comment]
You're welcome.  Those additional files/expressions can probably be removed from the `clean` target in the Makefile then as well.  Cheers.

**Pull request merged**